.github/json_matrices/build-matrix.json

```
@@ -2,16 +2,28 @@
     {
         "OS": "ubuntu",
         "NAMED_OS": "linux",
+        "OS_VERSION": "ubuntu-latest",
         "RUNNER": "ubuntu-latest",
         "ARCH": "x64",
         "TARGET": "x86_64-unknown-linux-gnu",
-        "PACKAGE_MANAGERS": ["pypi", "npm", "maven", "pkg_go_dev"],
+        "PACKAGE_MANAGERS": [],
         "run": "always",
         "languages": ["python", "node", "java", "go", "dotnet"]
     },
     {
         "OS": "ubuntu",
         "NAMED_OS": "linux",
+        "OS_VERSION": "ubuntu-20.04",
+        "RUNNER": "ubuntu-20.04",
+        "ARCH": "x64",
+        "TARGET": "x86_64-unknown-linux-gnu",
+        "PACKAGE_MANAGERS": ["pypi", "npm", "maven", "pkg_go_dev"],
+        "languages": ["go"]
```

```suggestion
        "PACKAGE_MANAGERS": ["pypi", "npm", "maven", "pkg_go_dev"],
        "languages": ["go", "java"]
```
Consider adding others too, otherwise CD for these clients will no-op